src/websocket/types.ts

```diff
@@ -82,10 +81,20 @@ export enum BotToVaicMessageName {
   playStreamStop = 'playStream.stop'
 }
 
+export enum MediaFormat {
```

Comment:
Add all.

Can also be something like:
```ts
export type MediaFormat =
  'raw/lpcm16' |
  'raw/lpcm16_24' |
  ...;
```
(not sure how this behaves with auto-completion)